src/views/register/index.vue

```diff
@@ -227,45 +289,41 @@ export default {
       template: '',
       templateGlobals: {},
 
-      checks: [
-        {
-          title: 'flows',
-          checked: false,
-          loading: false,
-        },
-        {
-          title: 'AI',
-          checked: false,
-          loading: false,
-        },
-        {
-          title: 'whatsapp_demo',
-          checked: false,
-          loading: false,
-        },
-      ],
+      checks: [],
+
+      organizationFormIsValid: false,
+      projectFormIsValid: false,
+      templateFormIsValid: false,
     };
   },
 
+  created() {
+    if (this.isCreatingOrgView) {
+      this.page = 'organization';
+    } else if (this.isCreatingProjectView) {
+      this.page = 'project';
+    }
+  },
+
   methods: {
-    ...mapActions(['updateProfile', 'addInitialInfo', 'createOrg']),
+    ...mapActions(['updateProfile', 'addInitialInfo']),
 
     filter,
 
-    nextPage() {
-      const pageIndex = this.pages.indexOf(this.page);
+    lowerCaseButFirst(sentence) {
+      return sentence.slice(0, 1) + sentence.slice(1).toLowerCase();
+    },
 
-      if (pageIndex + 1 !== this.pages.length) {
-        this.page = this.pages[pageIndex + 1];
-      } else {
+    goToNextPageOrSubmit() {
```

Comment:
`OnNextPageOrSubmit` might be a good way too

Reply:
When I see the prefix `on`, it refers me to an event. In this case, I think a `go` verb would be more appropriate: when the user clicks the button `goToNextPageOrSubmit`.